### DISCUSSION TITLE
PostgreSQL support?

### DISCUSSION THREAD

**Discussion by Viste** [azerothcore/azerothcore-wotlk#9884] [Ideas]
How about run core on PostgreSQL instead of MyShitQL?
I am too lazy to translate all the explanations, why? In English, so I will just say that on large setups with huge DB and many players it will work many times better than mysql + there are additional features that are not in mysql, such as TimescaleDB for time-series data.
If I see you're interested, I'll be able to detail the benefits of the action in more detail.
@FrancescoBorzi @Yehonal @Helias @Winfidonarleyan

**Comment by Winfidonarleyan** [comment]
Very interesting

**Comment by Helias** [comment]
It's interesting.

I don't know the benefits of postgres but if with large data it works better than mysql we could support it.

**Comment by FrancescoBorzi** [comment]
Yes, would be nice to support PostgreSQL **in addition to** MySQL (not instead). I remember MaNGOS used to support both back in the days, not sure why the support has been removed later on.

We'll need to add it to our CI as well so we make sure it doesn't break later.

**Comment by Viste** [comment]
i do some start work but its even cant be alpha maybe there will be PR in the spring

**Comment by chris102994** [comment]
Any plans in 2025 to implement this? Looking to run on k8s and postgres operator support is superior to mysql.

**Comment by scwall** [comment]
Hello, indeed, PostgreSQL would be much more suited to concurrent requests and the frequently encountered issues with deadlocks. We are opting for PostgreSQL in work due to its robustness in handling a large number of connections and transactions.

Thank's
